style.css

```diff
@@ -0,0 +1,264 @@
+/* css reset */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  
+  body {
+    font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+    line-height: 1.5;
+  }
+  
+  ul {
+    list-style: none;
+  }
+  
+  a {
+    text-decoration: none;
+  }
+  
+  #header {
+    display: flex;
+  }
+  
+  #nav-bar {
+    position: fixed;
+    top: 0px;
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+    background-color: rgb(255, 255, 255);
+    overflow: hidden;
+  }
+  /*
+  @media only screen and (max-width:600px){
+  #nav-bar ul {
+    display: flex;
+  }
+  #nav-bar ul li {
+    padding: 1rem 2rem;
+  }
+  #nav-bar ul li a {
+    color: #333;
+    font-size: 1.2rem;
+  }     } */
+  
+  /* UTILITY CLASSES */
```

Comment:
I'm sorry for the repetitions, there shouldn't be comments in your CSS code as this makes your codes look messy. Your codes need to look clean. Could you fix this?